src/lib/email-scrubber.js

```diff
@@ -0,0 +1,213 @@
+/* flow */
+/* eslint-disable no-console, no-param-reassign, no-mixed-spaces-and-tabs */
+
+import dns from 'dns';
+import { Socket } from 'net';
+import EventEmitter from 'events';
+import winston from 'winston';
+import callbackToPromise from './promisify';
+
+export class BulkEmailChecker extends EventEmitter {
```

Comment:
The class name and filename should match, either rename the file to `BulkEmailChecker` or rename the class to `EmailScrubber`